bin/strata-cli/src/cmd/change_pwd.rs

```diff
@@ -12,6 +13,17 @@ pub struct ChangePwdArgs {}
 pub async fn change_pwd(_args: ChangePwdArgs, seed: Seed, persister: impl EncryptedSeedPersister) {
     let term = Term::stdout();
     let mut new_pw = Password::read(true).unwrap();
+    let entropy = new_pw.entropy();
+    let _ = term.write_line(format!("Password strength (Overall strength score from 0-4, where anything below 3 is too weak): {}", entropy.score()).as_str());
```

Comment:
It's a nit, but I wouldn't even bother showing the user the score, since it's not really actionable on its own. If the passphrase is weak and there's feedback, let the user know. If it's strong, just set the passphrase.